app/src/main/java/edu/drexel/lapcounter/lapcounter/frontend/CurrentWorkoutActivity.java

```diff
@@ -219,8 +241,21 @@ protected void onStart() {
             return;
         }
 
+        if (wasCalibrating()) {
```

Comment:
this only seems to trigger when I re-calibrate a device. when I add a new device, I still see laps counted during calibration and no dialog.

Reply:
Pushed some commits. I wasn't able to test adding a new device since there's nothing around me that I can connect to besides good ol' RDL51822.